### DISCUSSION TITLE
Intel 8265 compatibility.

### DISCUSSION THREAD

**Discussion by UwUnyaa** [vanilla-wiiu/vanilla#42] [Hardware Results]
As requested, I'm moving my [issue](https://github.com/vanilla-wiiu/vanilla/issues/41) to a discussion, here's the body of the report in question:

Just checked that the software works on my Intel 8265 card. No surprises, it works as expected. Using a thinkpad T480, running arch.

Here's the relevant line from lspci:

    03:00.0 Network controller: Intel Corporation Wireless 8265 / 8275 (rev 78)

I'm not sure if I can edit the wiki as someone who hasn't contributed to the project.

**Comment by pepeshki** [comment]
This card is also used on the ThinkPad X280

**Comment by EnderShiru** [comment]
This card is also used in the Dell Latitude 7490 (in my model)

**Comment by haakjell** [comment]
This card is also used in the ThinkPad X380 Yoga

**Comment by TeraByte38** [comment]
This card is also used in the HP Spectre x360 13-ae (13t-ae seems to be the same; other numbers and letters after "ae" are the config)
edit: it works 👍
<img width="1920" height="1080" alt="1786190196751" src="https://github.com/user-attachments/assets/cbfa4681-d74e-4675-a29e-5c451ff93427" />
